src/modules/main/Main.tsx

```diff
@@ -29,10 +34,18 @@ const Main = ({ match }: RouteChildrenProps) => {
 
       <main>
         <Switch>
+          <Route
```

Comment:
I know you have implemented it based on the solution below. But this has a lot of disadvantages - we added a dependency to this file just to handle the redirect in case of a wrong address - this should be done inside the admin module. I will do the same for the Templates module later. Create hook for that before rendering or use <Redirect />